common/helpers/src/avro.rs

```diff
@@ -141,7 +141,7 @@ pub fn translate_schemas(serialized_schema: Vec<Vec<u8>>) -> Result<Vec<Schema>,
 /// let avro_schema = schema_result.unwrap().0;
 /// let schema_writer = avro::get_schema_data_writer(&avro_schema);
 /// ```
-pub fn get_schema_data_writer<'a>(schema: &'a Schema) -> Writer<'a, Vec<u8>> {
+pub fn get_schema_data_writer(schema: &Schema) -> Writer<Vec<u8>> {
```

Comment:
this is nothing to do with lint, can you please remove?

Reply:
All of my changes in this PR was required for provided lint rules to pass